Sources/BrowserServicesKit/SecureVault/SecureVaultManager.swift

```diff
@@ -167,16 +167,13 @@ extension SecureVaultManager: AutofillSecureVaultDelegate {
             }
 
             if delegate.secureVaultManagerIsEnabledStatus(self, forType: .password) {
-                getAccounts(for: domain,
-                            from: vault,
-                            or: passwordManager,
-                            withPartialMatches: includePartialAccountMatches) { [weak self] accounts, error in
+                getCredentials(forDomain: domain, from: vault, or: passwordManager) { [weak self] credentials, error in
```

Comment:
@graeme there is a regression here since `withPartialMatches: includePartialAccountMatches` has not been implemented when replacing `getAccounts` with `getCredentials` which is what allows subdomain matching to work e.g. 
- if you have credentials saved for `nytimes.com` 
- you visit the website [nytimes.com](https://www.nytimes.com/) 
- when you click to login you are redirected to `myaccount.nytimes.com`
- your `nytimes.com` credentials should be a suggestion since it’s the same TLD (nytimes.com)